lib/_http_outgoing.js

```diff
@@ -289,6 +310,34 @@ OutgoingMessage.prototype._buffer = function _buffer(data, encoding, callback) {
 };
 
```

Comment:
btw, it may be slightly more performant to use `Object.defineProperties()` to define these in a single call.